image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -95,15 +112,15 @@ Positive prompt: a beautiful pink unicorn
 
 Read the numpy latent input and noise for scheduler instead of C++ std lib for the alignment with Python pipeline.
 
-* Generate image with random data generated by Python: `./build/lcm_dreamshaper -r`
+* Generate image with random data generated by Python: `./build/Release/lcm_dreamshaper -r`
```

Comment:
`Release` is required only for multi config cmake generators like MSVC or Ninja Multi-Config
https://cmake.org/cmake/help/latest/prop_gbl/GENERATOR_IS_MULTI_CONFIG.html